run_project_tests.py

```diff
@@ -1393,6 +1399,13 @@ def tqdm_print(*args: mlog.TV_Loggable, sep: str = ' ') -> None:
         else:
             f.update_log(TestStatus.OK)
             passing_tests += 1
+            for cleanup_path in t.cleanup:
+                assert(not os.path.isabs(cleanup_path))
```

Comment:
Assert is not a function and this is a common cause of accidentally passing a tuple to it -- which is always true and thus defeats the purpose of an assert.